translate/storage/csvl10n.py

```diff
@@ -351,10 +351,10 @@ class csvfile(base.TranslationStore):
     Mimetypes = ['text/comma-separated-values', 'text/csv']
     Extensions = ["csv"]
 
-    def __init__(self, inputfile=None, fieldnames=None, encoding="auto"):
+    def __init__(self, inputfile=None, fieldnames=None, encoding=None):
         base.TranslationStore.__init__(self, unitclass=self.UnitClass)
         self.units = []
-        self.encoding = encoding or 'utf-8'
+        self.encoding = encoding or 'auto'
```

Comment:
Is it actually necessary to use `auto`? I mean, we should fallback to `utf-8` and that can be accomplished if encoding is `None`, isn't it?